File: PJDesignBackStageAPI/App.Model/Portfolio/GetPortfolioById.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace App.Model
{
    public class GetPortfolioByIdResponse : EditResponseBase
    {
        public int Id { get; set; }
        public bool IsBefore { get; set; }
        public string Title { get; set; } = null!;
        public string ThumbnailUrl { get; set; } = null!;
        public DateTime CreateDt { get; set; }
        public bool IsEnabled { get; set; }
        public DateTime? Date { get; set; }
        public List<Category>? Categories { get; set; }
        public List<string>? Photos { get; set; }
    }
}

File: PJDesignBackStageClient/src/app/features/portfolio/feature-shared/models/get-portfolio-by-id.ts
import { EditResponseBase } from "src/app/shared/models/bases";
import { Category } from "src/app/shared/models/category";

export class GetPortfolioByIdResponse extends EditResponseBase {
  id: number;
  isBefore: boolean;
  title: string;
  createDt: Date;
  isEnabled: boolean;
  date?: Date;
  categories: Category[] | null;
  photos?: string[]
}
